packages/phone-number-privacy/combiner/test/integration/pnp.test.ts

```diff
@@ -444,4 +454,192 @@ describe('pnpService', () => {
       })
     })
   })
+
+  describe(`${CombinerEndpoint.PNP_QUOTA}`, () => {
+    const useQuery = async (queryCount: number, signer: Server | HttpsServer) => {
+      for (let i = 0; i < queryCount; i++) {
+        const phoneNumber = '+1' + Math.floor(Math.random() * 10 ** 10)
+        const blindedNumber = getBlindedPhoneNumber(phoneNumber, BLINDING_FACTOR)
+        const req = {
+          account: ACCOUNT_ADDRESS1,
+          blindedQueryPhoneNumber: blindedNumber,
+        }
+        const authorization = getPnpRequestAuthorization(req, ACCOUNT_ADDRESS1, PRIVATE_KEY1)
+        await request(signer)
+          .post(SignerEndpoint.PNP_SIGN)
+          .set('Authorization', authorization)
+          .send(req)
+      }
+    }
+
+    const getCombinerQuotaResponse = async (req: PnpQuotaRequest, authorization: string) => {
+      const res = await request(app)
+        .get(CombinerEndpoint.PNP_QUOTA)
+        .set('Authorization', authorization)
+        .send(req)
+      return res
+    }
+
+    const totalQuota = 10
+    beforeAll(async () => {
+      let weiTocusd = new BigNumber(1e17)
+      mockOdisPaymentsTotalPaidCUSD.mockReturnValue(weiTocusd.multipliedBy(totalQuota))
+    })
+
+    const queryCountParams = [
+      [0, 0, 0, 0],
```

Comment:
Nit: While I do appreciate the minimalism here, this might be slightly more readable if you used an array of objects with named fields. At least, you could use a named field for the expectedQueryCount and keep an array for the signerQueries

Reply:
Up to you though. If you think this is cleaner feel free to keep it :)

Reply:
sure, changed to array of objects :)